tests/ignite/distributed/utils/__init__.py

```diff
@@ -228,6 +228,32 @@ def _test_distrib_barrier(device):
     assert tt.item() == true_res + 10.0
 
 
+def _test_distrib_new_group(device):
```

Comment:
You does not call anywhere this test ?
Please also add a subtest for `idist.new_group` in non-distributed config